packages/devextreme/js/core/component.js

```diff
@@ -82,6 +83,8 @@ export const Component = Class.inherit({
         this._disposingCallbacks = _disposingCallbacks || Callbacks();
         this.postponedOperations = new PostponedOperations();
         this._createOptions(options);
+
+        license.verifyLicense(Config().license);
```

Comment:
There is no test for obtaining license token from config. We discussed this with @ksercs and I agree with his proposal: we can try moving `Config().license` inside `verifyLicense` and mocking it `Config().License`

Is this possible?

Reply:
Unclear how to mock default exported function in tests.

Reply:
Maybe mocking in this case is not good